tokio-console/src/config.rs

```diff
@@ -394,33 +440,34 @@ impl ConfigFile {
         Ok(config)
     }
 
-    fn into_view_options(self) -> ViewOptions {
-        ViewOptions {
-            no_colors: self.no_colors(),
-            lang: self.charset.as_ref().and_then(|config| config.lang.clone()),
-            ascii_only: self.charset.as_ref().and_then(|config| config.ascii_only),
-            truecolor: self.colors.as_ref().and_then(|config| config.truecolor),
-            palette: self.colors.as_ref().and_then(|config| config.palette),
-            toggles: ColorToggles {
-                color_durations: self.color_durations(),
-                color_terminated: self.color_terminated(),
-            },
-        }
+    fn target_addr(&self) -> color_eyre::Result<Option<Uri>> {
+        let uri = self
+            .target_addr
+            .as_ref()
+            .map(|addr| addr.parse::<Uri>())
+            .transpose()
+            .wrap_err(format!("failed to parse Uri, {:?}", self.target_addr))?;
```

Comment:
nit: let's prefer to use `wrap_err_with` when we're emitting a formatted `String`, as `wrap_err_with` will avoid allocating to format the string when no error occurs
```suggestion
            .wrap_err_with(|| format!("failed to parse Uri, {:?}", self.target_addr))?;
```

Reply:
also, i might change the message a bit:
```suggestion
            .wrap_err_with(|| format!("failed to parse target address {:?} as URI", self.target_addr))?;
```

Reply:
I fixed: https://github.com/tokio-rs/console/pull/334/commits/ded150295c5abfcfd49cba51a321fff3fa39ca28

> nit: let's prefer to use wrap_err_with when we're emitting a formatted String, as wrap_err_with will avoid allocating to format the string when no error occurs

I learned, thank you!